### PR TITLE
initial commit for app sdk docs AM-979

### PR DESCRIPTION
**Pull request by jryoo**

### CODE REVIEW

File: docs/sdk-beta/beta-signup.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# DroneDeploy App SDK BETA signup
+
+Dear DroneDeploy Developer,
+
+With over **70 apps** on the App Market ranging from requesting flights on demand to counting plants, it is clear that our developers are incredibly creative and continue to push the boundaries of what apps can do on the DroneDeploy platform.
+
+
+At DroneDeploy we are committed to our developers and want to give you the tools necessary to build what you want to. That is why we are excited to announce the next generation of DroneDeploy’s App Platform, with 3 innovative platform features:
```
Comment (ckoberlein) [inline docs/sdk-beta/beta-signup.md]: Technically `build what you want to` (ends in a preposition) should be `build what you want`.

File: docs/sdk-beta/datastore.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Datastore - COMING SOON
```
Comment (ckoberlein) [inline docs/sdk-beta/datastore.md]: Is there more information to be added?  If not, should we remove `COMING SOON`?

File: docs/sdk-beta/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+# Getting Started
+
+## Pre-requisites
```
Comment (ckoberlein) [inline docs/sdk-beta/getting-started.md]: Should be `Prerequisites`

File: docs/sdk-beta/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+## Sample App
+
+To get you started, we have created a sample app. By deploying this sample app, you will learn to use every aspect of the DroneDeploy SDK.
+
+### Overview
+
+The app you will be deploying will enable the following flow:
+1. A user creates an export
+1. When the export completes, an export complete Trigger will go off
```
Comment (ckoberlein) [inline docs/sdk-beta/getting-started.md]: Trigger is capitalized here, and not on the line below.  I'd say make them the same for consistency.

File: docs/sdk-beta/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+1. The URL should look something like this
+
+        https://maker.ifttt.com/trigger/dronedeploy/with/key/calBarcsa1DcqmSN9_D1acb
+
+1. Copy and paste your URL into your App UI and press save. This will save this URL by passing the URL string to the `ifttt-webhook` function, which will then save the URL to the defined `webhook-table` Datastore table. Try refreshing the page and opening the app - you will notice that your URL is still there and now persisted in your Datastore!
+
+    ![](ifttt-sample-save.gif)
+
+1. Now if you go and do a DroneDeploy map export, you should get an event in IFTTT!
```
Comment (ckoberlein) [inline docs/sdk-beta/getting-started.md]: Could change `go and do` to `perform`

File: docs/sdk-beta/introduction.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+Welcome to the DroneDeploy App SDK Beta. We are very excited to share with you what we have been working on to help you make the most out of the DroneDeploy Platform. By participating in this beta, you will have access to the latest features of the DroneDeploy Platform and also have an impact and influence over the product roadmap.
+
+The new App SDK introduces three main features in addition to the existing UI platform:
+
+![](overview.png)
+
+
+* [**UI**](../getting_started.md): Build a custom user interface
+
+* [**Functions**](functions.md): Serverless architecture running on the DroneDeploy Platform. Write Node.js code to enable backend functionality.
```
Comment (ckoberlein) [inline docs/sdk-beta/introduction.md]: These descriptions for Functions, Datastore and Triggers are different than those on the `beta-signup` page.  Is that intentional?

File: docs/sdk-beta/datastore.md (hunk 2 above)
Comment (jryoo) [reply docs/sdk-beta/datastore.md]: yes will come in next PR going out today

File: docs/sdk-beta/introduction.md (hunk 6 above)
Comment (jryoo) [reply docs/sdk-beta/introduction.md]: yes intentional

**Pull request merged**